tests/unit/store/flask.spec.js

```diff
@@ -398,16 +402,18 @@ describe("store/flask", () => {
         const bound_ping_system_status = ping_system_status.bind(context);
         await bound_ping_system_status();
 
-        expect(store.state.flask.status_uuid).toEqual(
+        expect(store.state.flask.status_uuid).toStrictEqual(
           STATUS.MESSAGE.CALIBRATION_NEEDED
         );
 
-        expect(store.state.playback.playback_state).toEqual(
+        expect(store.state.playback.playback_state).toStrictEqual(
           playback_module.ENUMS.PLAYBACK_STATES.CALIBRATION_NEEDED
         );
-        expect(store.state.playback.barcode).toEqual(invalid_plate_barcode);
+        expect(store.state.playback.barcode).toStrictEqual(
+          invalid_plate_barcode
+        );
       });
-      test("Given the current state is SERVER_READY and the returned state is CALIBRATION_NEEDED with a <empty> plate bar code value, Then the  Playback State should update to CALIBRATION_NEEDED and the plate was removed so the value of plate_barcode was rest to `null` as  from JSON object contained <empty>", async () => {
+      test("Given that the /system_status is mocked with a response of CALIBRATION_NEEDED, When the ping_system_status is active, Then the  Playback State should update to CALIBRATION_NEEDED and the plate was removed so the value of plate_barcode was rest to `null` as  from JSON object contained <empty>", async () => {
```

Comment:
typo 'rest'->'reset'.  also, the concept that "as the JSON object contained an empty string" is part of the setup of the test. It should be in the "Given" section describing the mocked HTTP response, not the "Then" section

Reply:
Corrected.